### PR TITLE
added styles for loading spinner

### PR DESCRIPTION
**Pull request by pospyra**
https://trello.com/c/RRiW1DBx/128-loader

### CODE REVIEW

**Review by SashaBondarchuk — APPROVED** [review]

File: frontend/src/app/shared/components/loading-spinner/loading-spinner.component.html [hunk 1]
```diff
@@ -1,12 +1,4 @@
-<div *ngIf="spinnerService.isLoading$ | async">
-    <div
-        [ngStyle]="{
-            'font-size': size,
-            margin: margin,
-            top: top,
-            left: left,
-            position: position
-        }"
```
Comment (tatianahutii) [inline frontend/src/app/shared/components/loading-spinner/loading-spinner.component.html]: why did you remove this?

File: frontend/src/app/shared/components/loading-spinner/loading-spinner.component.html [hunk 2]
```diff
@@ -1,12 +1,4 @@
...
-        class="loader"
-    ></div>
+<div *ngIf="spinner.isLoading$ | async">
+    <div class="loader"></div>
+    <p class="loader-text">Loading...</p>
```
Comment (tatianahutii) [inline frontend/src/app/shared/components/loading-spinner/loading-spinner.component.html]: we don't need this is the spinner component

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/src/app/shared/components/loading-spinner/loading-spinner.component.html (hunk 1 above)
Comment (pospyra) [reply frontend/src/app/shared/components/loading-spinner/loading-spinner.component.html]: styles are set in sass file 
why do we need this here?

File: frontend/src/app/shared/components/loading-spinner/loading-spinner.component.html (hunk 2 above)
Comment (pospyra) [reply frontend/src/app/shared/components/loading-spinner/loading-spinner.component.html]: you mean we don’t need this text at all or move it to another place?

File: frontend/src/app/shared/components/loading-spinner/loading-spinner.component.html (hunk 1 above)
Comment (pospyra) [reply frontend/src/app/shared/components/loading-spinner/loading-spinner.component.html]: and because it is impossible to use our color-variables in ts

File: frontend/src/app/shared/components/loading-spinner/loading-spinner.component.html (hunk 2 above)
Comment (pospyra) [reply frontend/src/app/shared/components/loading-spinner/loading-spinner.component.html]: according to the task- "Loader should  be interactive"

File: frontend/src/app/shared/components/loading-spinner/loading-spinner.component.html (hunk 1 above)
Comment (tatianahutii) [reply frontend/src/app/shared/components/loading-spinner/loading-spinner.component.html]: in case when we have styles in a style file, this component is not reusable and cannot be customized for different needs

File: frontend/src/app/shared/components/loading-spinner/loading-spinner.component.html (hunk 2 above)
Comment (tatianahutii) [reply frontend/src/app/shared/components/loading-spinner/loading-spinner.component.html]: yes, we don't need this text at all

Comment (pospyra) [reply frontend/src/app/shared/components/loading-spinner/loading-spinner.component.html]: removed

File: frontend/src/app/shared/components/loading-spinner/loading-spinner.component.html (hunk 1 above)
Comment (pospyra) [reply frontend/src/app/shared/components/loading-spinner/loading-spinner.component.html]: ok, returned

**Review by tatianahutii — APPROVED** [review]

**Review by apofis96 — APPROVED** [review]
